core/pom.xml

```diff
@@ -109,6 +109,14 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
```

Comment:
Not only no, but hell no =)

Core is meant to be java 8 compatible, please rollback

Reply:
That was unintentional and unnecessary, I'm very glad you caught it.